extensions/eclipse-che-theia-plugin-ext/src/node/che-product-service.ts

```diff
@@ -87,8 +87,14 @@ export class CheProductServiceImpl implements CheProductService {
         } else {
             // relative path
             const productJsonDir = path.dirname(productJsonPath);
-            return 'file://' + path.join(productJsonDir, resource);
+            return path.join(productJsonDir, resource);
         }
     }
 
+    protected static toBase64(file: string): string {
+        const content = fs.readFileSync(file);
```

Comment:
What about http resources? Did you test it?

Reply:
Is it documented somewhere what types of resources we support in `PRODUCT_JSON`

Reply:
https://www.eclipse.org/che/docs/che-7/branding-che-theia/

Reply:
There also was a discussion to make it possible to use remote product.json.
```
export PRODUCT_JSON='http://some.host/path/product.json'
```

Reply:
If it's a http resource, we don't read it's content. Just give the URI to the plugin, it should work.

Reply:
Thanks @vitaliy-guliy for pointing this out.
I reworked it with the latest commit and checked. Works well.